### ISSUE TITLE
Simulator Fails From Slot 10 And Beyond

### ISSUE THREAD

**Issue by nisdas** [prysmaticlabs/prysm#741]
Currently if the simulator is left running , from slot 10 on wards it starts logging this error 
```go
[2018-11-12 01:37:53] ERROR simulator: Failed to get shard committee: slot 10 out of bounds: 0 <= slot < 10
[2018-11-12 01:37:55] ERROR simulator: Failed to get shard committee: slot 11 out of bounds: 0 <= slot < 10
[2018-11-12 01:37:57] ERROR simulator: Failed to get shard committee: slot 12 out of bounds: 0 <= slot < 10
```
This issue has to do when retrieving committees from the crystallized state for the simulator 
```go
	committees, err := cState.GetShardsAndCommitteesForSlot(parentSlot)
	if err != nil {
	       log.Errorf("Failed to get shard committee: %v", err)
	       continue
	}
```
A solution for this would be to modify the crystallized state to account for slots beyond the 9th slot

**Comment by terenc3t** [comment]
closed via #744

**Comment by rawfalafel** [comment]
Can you explain how this is reproduced? I ran the beacon chain simulator and couldn't reproduce.
